docs/src/config.ts

```diff
@@ -70,4 +70,16 @@ export const sidebar = [
 
 export const site = {
   title: 'Astro Documentation',
+  description: 'Build faster websites with less client-side Javascript',
+  ogLocale: 'en_US',
+  image: {
+    src: '/default-og-image.png?v=1',
```

Comment:
twitter requires this to be a full URL, including `https://`

```suggestion
    src: 'https://docs.astro.build/default-og-image.png?v=1',
```